File: VB/MainDemo.Module/MainDemoModule.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Security.Adapters
Imports DevExpress.ExpressApp.Security.Xpo.Adapters

Namespace MainDemo.[Module]
    Public NotInheritable Partial Class MainDemoModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
            IsGrantedAdapter.Enable(New XpoIntegratedCachedRequestSecurityAdapterProvider(), ReloadPermissionStrategy.CacheOnFirstAccess)
        End Sub
    End Class
End Namespace

File: CS/MainDemo.Module/MainDemoModule.cs
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Security.Adapters;
using DevExpress.ExpressApp.Security.Xpo.Adapters;

namespace MainDemo.Module {
	public sealed partial class MainDemoModule : ModuleBase {
		public MainDemoModule() {
			InitializeComponent();
			IsGrantedAdapter.Enable(new XpoIntegratedCachedRequestSecurityAdapterProvider(), ReloadPermissionStrategy.CacheOnFirstAccess);
		}
	}
}
